watertap/costing/unit_models/ion_exchange.py

```diff
@@ -336,9 +336,11 @@ def cost_ion_exchange(blk):
             to_units=blk.costing_package.base_currency,
         )
     )
+    blk.costing_package.add_cost_factor(blk, "TIC")
     blk.capital_cost_constraint = pyo.Constraint(
         expr=blk.capital_cost
-        == pyo.units.convert(
+        == blk.cost_factor
```

Comment:
There is currently a ``total_installed_cost_factor`` with value of 1.65 used in capital costs for IX that I had included before there was a global ``TIC``. That should be removed entirely and use the global variable to not be double counting.

Reply:
Of course whatever you decide to do it will cause the IX test to fail.